### PR TITLE
Support Snapshot of multiple tables

### LINKED ISSUE #6333: Support mutli-table in take snapshot
The take-spanshot command takes multiple keyspaces it would be helpful to support multiple tables as well.

### PR DESCRIPTION
**Pull request by amnonh**
This series adds support for taking a snapshot of multiple tables.

Fixes #6333

### CODE REVIEW

File: service/storage_service.cc [hunk 1]
```diff
@@ -2076,30 +2076,37 @@ future<> storage_service::take_snapshot(sstring tag, std::vector<sstring> keyspa
 
 }
 
-future<> storage_service::take_column_family_snapshot(sstring ks_name, sstring cf_name, sstring tag) {
+future<> storage_service::take_column_family_snapshot(sstring ks_name, std::vector<sstring> tabels, sstring tag) {
```
Comment (avikivity) [inline service/storage_service.cc:2079]: s/tabels/tables/

File: api/storage_service.cc [hunk 2]
```diff
@@ -992,12 +992,12 @@ void set_snapshot(http_context& ctx, routes& r) {
 
     ss::take_snapshot.set(r, [](std::unique_ptr<request> req) {
         auto tag = req->get_query_param("tag");
-        auto column_family = req->get_query_param("cf");
+        auto column_families = split(req->get_query_param("cf"), ",");
```
Comment (avikivity) [inline api/storage_service.cc:996]: We should have a separate REST endpoint that takes a list.

Comment (amnonh) [reply api/storage_service.cc:996]: In the new version of the API, definitely, but this is in sync with how the API looks like today for all other column_family commands and it was a request to keep the naming convention

File: service/storage_service.cc (hunk 1 above)
Comment (amnonh) [reply service/storage_service.cc:2079]: I'll fix and resend

**Comment by amnonh** [comment]
@avikivity I have forced push a new version with the typo fixed

**Comment by slivne** [comment]
@avikivity ping

File: api/storage_service.cc (hunk 2 above)
Comment (avikivity) [reply api/storage_service.cc:996]: ok (yuch)

**Pull request merged**